.firebaserc

```diff
@@ -1,7 +1,7 @@
 {
   "projects": {
     "web": "celebrityfanalizer",
-    "default": "celebrityfanalizer"
+    "default": "celebrityfanalyzer-dev"
```

Comment:
@sana7044, I didn't get this.

Reply:
It specifies project ID. This means in which project the functions will be deployed.

Reply:
I know, I am wondering why we are using the `dev` project now.

Reply:
I have access to the CelebrityFanalyzar-Dev project and when I was trying to deploy the cloud functions I would need to specify in which project I needed to deploy those functions so I used dev. In my next commit, I will remove it.